### DISCUSSION TITLE
reedem cüzdan kısmı hatası

### DISCUSSION THREAD

**Discussion by mina2028** [IBAX-io/go-ibax#1590] [Soru Cevap - Türkçe]
arkadaşlar cüzdanı bağlamak için redeem kısmına girilmiyor sayfa bulunamadı diyor. Bu konuda yardımcı olabilir misiniz ?

**Comment by Yusufknk** [comment]
Sayfa gelişim aşamasında olduğu için bazi zamanlar bakım çalışması olacaktır. Reedem bölümüne geldiğiniz zaman lütfen biraz bekleyin önceden cüzdan bağladıysanız görünecektir.
